src/main.cpp

```diff
@@ -510,10 +510,10 @@ game_value exportMapCommand(game_state& gs, SQFPar rightArg) {
 }
 
 void intercept::pre_start() {
-    static auto grad_me_export_map =
+    static auto grad_meh_export_map =
         client::host::register_sqf_command("gradMeExportMap", "Exports the given map", exportMapCommand, game_data_type::BOOL, game_data_type::STRING);
```

Comment:
gradMe**h**ExportMap?

Reply:
Far better than `grad_map_export_export_map` 😅

Reply:
Fixed in https://github.com/gruppe-adler/grad_me/pull/1/commits/55cd8022cd3019e5acdb7594bb9b096573bfd8d7